### PR TITLE
Mobile Skeleton

### PR DESCRIPTION
**Pull request by sj1978**
in this Project I:
. created an index.html
. created a style.css file
.added tags and style
.added linters 
.added a readme and mit file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <nav class="menu">
+            <div class="nav-div"> Jhon Doe</div>
```
Comment (e71az) [inline index.html:18]: - [ ] Please align this text to be at the same level as the hamburger button to comply with the project requirements:
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            </svg>
+
+        </nav>
+    </header>
+    <main class="content">
+        <h1>Anibal Amoroso</h1>
+        <h2>Software Developer</h2>
+        <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If
+            you like what
+            you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (e71az) [inline index.html:30]: - [ ] The container of the section is a section tag (Please wrap this inside a section tag)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <header>
+        <nav class="menu">
+            <div class="nav-div"> Jhon Doe</div>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+
+        </nav>
+    </header>
+    <main class="content">
```
Comment (e71az) [inline index.html:25]: - [ ] You are missing this whole section, please add it and don't forget that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag):
image

**Review by e71az — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @sj1978, I am Elias, your reviewer for this PR

Well done so far with your project!

Your project is looking great so far in comparison with the design given in Figma!

Please correct the issues mentioned in the comments section and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @sj1978 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

## Optional suggestions 👌 

- [ ] [OPTIONAL] Please make sure you write the commit messages in the imperative tense. [Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) you can read about the formatting rules. Remember to use this practices in the futures commits. 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**